webpack-config.js

```diff
@@ -28,17 +25,46 @@ module.exports = {
       },
     }],
   },
+  resolve: {
+    extensions: ['.ts', '.js', '.mjs', '.tsx', '.jsx', '...'],
+    alias: {
+      client: clientDir,
+    },
+  },
+};
+
+const clientConfig = {
+  ...baseConfig,
+  name: 'client',
+  entry: clientRootFile,
+  output: {
+    path: outputScriptsDir,
+    filename: 'app.js',
+  },
+};
```

Comment:
It may make sense to split these configurations out into more modular files in the future: cleaner separation of development/production settings as well as client/server settings.

Reply:
Created a TODO list in a comment on issue #17 to cover this